test.js

```diff
@@ -0,0 +1,381 @@
+import test from 'ava';
+import 'babel-core/register';
```

Comment:
was confused about this in ava, weird that it doesn't support importing es6 out of the box